packages/app-store/hubspotothercalendar/lib/CalendarService.ts

```diff
@@ -38,6 +39,22 @@ export default class HubspotOtherCalendarService implements Calendar {
     this.log = logger.getChildLogger({ prefix: [`[[lib] ${this.integrationName}`] });
   }
 
+  private hubspotContactCreate = async (attendees: Person[]) => {
+    const simplePublicObjectInputs: SimplePublicObjectInput[] = attendees.map((attendee) => {
+      const [firstname, lastname] = attendee.name ? attendee.name.split(" ") : [attendee.email, ""];
+      return {
+        properties: {
+          firstname,
+          lastname,
+          email: attendee.email,
+        },
+      };
+    });
+    return Promise.all(
+      simplePublicObjectInputs.map((contact) => hubspotClient.crm.contacts.basicApi.create(contact))
+    );
```

Comment:
Although there is a Batch API for contact creation, there is a delay between the creation and having the contacts present to search through the API to continue with the logic of meeting creation and the association between meeting and contacts. Luckily, the simple contact create API returns the created contact information, so we create all contacts and wait till all are done to continue with the described flow to assure every contact gets the meeting.